src/frontend/screens/AppPasscode/TurnOffPasscode.tsx

```diff
@@ -122,24 +156,27 @@ export const TurnOffPasscode = ({ setScreenState }: TurnOffPasscodeProps) => {
           sheetRef.current?.close();
         }}
       />
-    </React.Fragment>
+    </View>
   );
 };
 
-interface ConfirmTurnOffPasswordModal {
+interface ConfirmTurnOffPasswordModalProps {
   turnOffPasscode: () => void;
   closeSheet: () => void;
 }
 
 const ConfirmTurnOffPasswordModal = React.forwardRef<
   BottomSheetMethods,
-  ConfirmTurnOffPasswordModal
+  ConfirmTurnOffPasswordModalProps
 >(({ turnOffPasscode, closeSheet }, sheetRef) => {
   const [snapPoints, setSnapPoints] = React.useState<(number | string)[]>([
     0,
     "40%",
   ]);
 
+  // This is crude, but it is suddenly opening and then closing if I don't do this...
+  React.useLayoutEffect(() => closeSheet(), [closeSheet]);
+
```

Comment:
Without this, it is flickering open and closing right away when navigated to this screen. I originally thought it was because I wasn't using native navigation to navigate here, which lead me to add all these pages to native navigation....but that wasn't the problem (but we should probably use native navigation anyway....)